### PR TITLE
[REL] Migration website_doc to 11.0

### PR DESCRIPTION
**Pull request by nicomacr**
* [FIX] update Readme & manifest
* [REM] js code unused
* [FIX] name of .xml views files
* [FIX] separe into to files .py the clases according guidelines

### CODE REVIEW

File: website_doc/controllers/main.py [hunk 1]
```diff
@@ -6,10 +6,7 @@
 from odoo.http import request
 import logging
 import werkzeug.utils
-# from odoo.addons.web import http
-# from odoo.addons.web.http import request
 from odoo.addons.website.controllers.main import Website as controllers
-# from odoo.addons.website.models.website import slugify
 
 logger = logging.getLogger(__name__)
```
Comment (zaoral) [inline website_doc/controllers/main.py]: remove not used logger, please remove both logger var and logging import, apply this to all python files in the module

**Review by zaoral — CHANGES_REQUESTED** [review]
Hi @nicomacr 

I have made a quick review.
Tomorrow will make a more extensive one.

* Please fix the next pylint errors https://travis-ci.org/ingadhoc/website/jobs/410003559#L836-L843
* Runbot and travis failing to run because they do not found the base_name_search_improve, should we update oca_dependencies? with oca/server-tool or ingadhoc/patches

File: website_doc/controllers/main.py (hunk 1 above)
Comment (jjscarafia) [reply website_doc/controllers/main.py]: Use ingadhoc/patches for now...

File: oca_dependencies.txt [hunk 2]
```diff
@@ -4,3 +4,4 @@ ingadhoc-odoo-argentina https://github.com/ingadhoc/odoo-argentina.git
 # ingadhoc-argentina-sale https://github.com/ingadhoc/argentina-sale.git
 # ingadhoc-reporting-engine https://github.com/ingadhoc/reporting-engine.git
 oca-sale_workflow https://github.com/OCA/sale-workflow.git
+ingadhoc-patches https://github.com/ingadhoc/patches.git
```
Comment (zaoral) [inline oca_dependencies.txt]: I think that a comment with a todo should be added here in order to take into account when we merge base_name_search_mproved to oca/server-tools

File: website_doc/controllers/website_search.py [hunk 3]
```diff
@@ -4,14 +4,10 @@
 ##############################################################################
 from odoo import http
 from odoo.http import request
-import logging
-
 from odoo.tools.translate import _
-from odoo.tools import html2text
+import html2text
```
Comment (zaoral) [inline website_doc/controllers/website_search.py]: Please need to use the next convention avoid travis error
```python
try:
    from mercadopago import mercadopago
except ImportError:
    _logger.debug('Cannot import external_dependency mercadopago')
```

**Review by zaoral — CHANGES_REQUESTED** [review]
Hi @nicomacr 
New technical review
Please review this suggestions

**Pull request merged**